### PR TITLE
Use pod utilities for CAPA CI conformance presubmit job

### PR DESCRIPTION
**Pull request by dims**
Also switch pull-cluster-api-provider-gcp-make to use `preset-kind-volume-mounts` while we are here

### CODE REVIEW

File: config/jobs/kubernetes-sigs/cluster-api-provider-gcp/cluster-api-provider-gcp-presubmits.yaml [hunk 1]
```diff
@@ -73,25 +59,22 @@ presubmits:
...
-            - "--repo=sigs.k8s.io/kind=master"
-            - "--service-account=/etc/service-account/service-account.json"
-            - "--upload=gs://kubernetes-jenkins/pr-logs"
-            - "--scenario=execute"
-            - "--"
-            - "bash"
-            - "--"
-            - "-c"
-            - "cd ./../../sigs.k8s.io/cluster-api-provider-gcp && scripts/ci-e2e.sh"
+        - image: gcr.io/k8s-testimages/kubekins-e2e:v20191103-6816af1-experimental
```
Comment (detiber) [inline config/jobs/kubernetes-sigs/cluster-api-provider-gcp/cluster-api-provider-gcp-presubmits.yaml]: Is experimental actually needed here?

Comment (dims) [reply config/jobs/kubernetes-sigs/cluster-api-provider-gcp/cluster-api-provider-gcp-presubmits.yaml]: picked up the same image as used in `pr-make` as that job seems to be working well.

Comment (detiber) [reply config/jobs/kubernetes-sigs/cluster-api-provider-gcp/cluster-api-provider-gcp-presubmits.yaml]: gotcha, I want to say we originally switched to the experimental images when we needed a new version of bazel, so we can likely switch back, but that can be done separately.

**Pull request merged**